src/main/resources/static/gogo_components/pokemon-view/pokemon-view.html

```diff
@@ -14,7 +14,8 @@
             <div class="thumbnail">
                 <img src="https://img.pokemondb.net/artwork/{{pokemon.pokedexLink}}.jpg" style="height: 200px;"/>
                 <div class="caption">
-                    <h3>{{pokemon.name}}</h3>
+                    <h3>{{pokemon.nickname}}</h3>
```

Comment:
I was thinking that maybe in order not to explode the box. We could always display the nickname (which in most cases will the just the name of the pokemon unless you're Afonso and rename all your pokemons). And only in case the nickname and name are different display the name of the pokemon below?

Cause otherwise for most of my pokemon I will have duplicates right now.